metric_learn/base_metric.py

```diff
@@ -177,9 +240,87 @@ def transform(self, X):
                              accept_sparse=True)
     return X_checked.dot(self.transformer_.T)
 
+  def get_metric(self):
+    """Returns a function that takes as input two 1D arrays and outputs the
```

Comment:
It would be nice to only write this docstring once. Unfortunately Python doesn't do inherited docstrings, so we have to assign it manually:

```
def get_metric(self):
  transformer_T = ...  # code here as usual

get_metric.__doc__ = BaseMetricLearner.get_metric.__doc__
```

Reply:
Cool thanks ! I always wondered how to avoid copying docstrings